extensions/interactions/MultipleChoiceInput/directives/multiple-choice-input-validation.service.spec.ts

```diff
@@ -187,6 +187,10 @@ describe('MultipleChoiceInputValidationService', () => {
       // default outcome.
       goodAnswerGroups[0].rules.splice(1, 1);
       warnings = validatorService.getAllWarnings(
+        // This throws "Type 'null' is not assignable to type 'Outcome'
+        // ." We need to suppress this error because of the need to test
+        // validations when the outcome is not defined.
+        // @ts-ignore
```

Comment:
If we are checking for this, than it is a valid input to the function and should be added as part of the type def of that argument.

Reply:
done.